lib/modules/datasource/docker/index.ts

```diff
@@ -913,6 +913,11 @@ export class DockerDatasource extends Datasource {
     return digest;
   }
 
+  @cache({
+    namespace: 'datasource-docker-hub-tags',
+    key: (registryHost: string, dockerRepository: string) =>
```

Comment:
Are you sure that `registryHost` is not available in that context?
Also this function is only used for DocherHub, so there is only one host.

Reply:
yes, sorry, will fix